src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
+  root: {
+    width: "1100px",
+    margin: "0 auto",
+    background: "white",
+    padding: "0 5em 5em",
+  },
+  arrowBody: {
+    width: "300px",
+    marginLeft: "20px",
+    background: "#C9CED2",
+    padding: "5px 20px",
+  },
+  triangle: {
+    width: 0,
+    height: 0,
+    background: "white",
+    borderTop: "30px solid transparent",
+    borderBottom: "30px solid transparent",
+    borderLeft: "30px solid #C9CED2",
+  },
+  dropdown: {
+    width: "300px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    borderRadius: "500px",
+    background: "#F2F5F7",
+  },
+  centerRow: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  third: {
+    width: "300px",
+    height: "50px",
+    margin: "30px 15px",
+  },
+  half: {
+    width: "450px",
+    height: "50px",
+    margin: "30px",
+  },
```

Comment:
Hardcoding sizes like this is pretty fragile. If you want something to be 50% width, you just just set the container to `display: flex` and the buttons inside to have `flex: 1` so that they automatically take up half the containers width each. That way, if a button is added or removed, you dont have to change anything, whereas with this you have to go around and update all these sizes manually.

In addition, instead of setting the height manually, you can use the `size` prop and set it to `size="large"`